cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/startup/WorkerApplication.java

```diff
@@ -107,10 +109,8 @@ protected void configure() {
             	}
             	bind(bucketName).named("config:se.simonsoft.cms.publish.bucket").to(String.class);
             	cloudId = environment.getParamOptional("CLOUDID");
-
-            	region = Region.getRegion(Regions.fromName("eu-west-1")); // Currently hardcoded, might need different regions annotated per service.
-            	// Might need to determine which region we are running in for EC2. See SDK 2.0,
-            	// might have a Region Provider Chain.
+            	region = RegionUtils.getRegion(new DefaultAwsRegionProviderChain().getRegion());
+            	logger.debug("Region name: {}", region.getName());
```

Comment:
For safety (and backwards compat), add check for region == null with fallback to "eu-west-1" (and log an info statement).

Reply:
Done!